app/parsers/bulkrax/parser_export_record_set.rb

```diff
@@ -87,14 +87,14 @@ def each
       #
       # @see #file_sets
       def candidate_file_set_ids
-        @candidate_file_set_ids ||= works.flat_map { |work| work.fetch("file_set_ids_ssim", []) }
+        @candidate_file_set_ids ||= works.flat_map { |work| work.fetch("#{Bulkrax.file_model_name.to_s.underscore}_ids_ssim", []) }
       end
 
-      # @note Specifically not memoizing this so we can merge values without changing the object.
+      # @note Specifically not memorizing this so we can merge values without changing the object.
```

Comment:
```suggestion
      # @note Specifically not memoizing this so we can merge values without changing the object.
```

Reply:
I bet that this was my spellchecker. Never works when it is supposed to, changes things when it shouldn't